docs/UserGuide.md

```diff
@@ -175,24 +176,41 @@ If your changes to the data file makes its format invalid, FinBook will discard
 
 ---
 
-### Importing data from external sources [coming soon]
+### Importing data : `import`
+
+Imports data from a `JSON` or `CSV` file
+
+* `JSON` files must be saved by the latest version of FinBook
+* `CSV` files must be exported as Google CSV from Google Contacts
+
+Format: `import PATH`
+
+* Imports data from the file at the specified `PATH`
+* `PATH` can be a relative or full path
+* `PATH` must end in `.json` or `.csv`
+
+Examples:
+
+* `import ./data.json` imports data from the file `data.json` which is located in the same directory as the FinBook executable
+* `import ../data.csv` imports data from the file `data.csv` which is located one level outside the directory of the FinBook executable
```

Comment:
I feel that the example here should be consistent with our string `MESSAGE_USAGE` in `ImportCommand.java`